lighthouse/blueprints/plates.py

```diff
@@ -128,6 +128,21 @@ def find_plate_from_barcode() -> FlaskResponse:
         ]
     }`
 
+    `GET /plates?barcodes=destination_123,destination_456&_type=destination`
+
+    `{"plates":
+        [
+            {
+                "plate_barcode": "destination_123",
+                "plate_exists": true,
+            },
+            {
+                "plate_barcode": "destination_456",
+                "plate_exists": false,
+            },
+        ]
+    }`
```

Comment:
Does it work to wrap multiple lines in single backticks?  I think you need to have three backticks to form a fence each side of the code block.

Reply:
Thanks, I've added a bit more formatting now.